physical/s3_test.go

```diff
@@ -0,0 +1,78 @@
+package physical
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/awslabs/aws-sdk-go/aws"
+	"github.com/awslabs/aws-sdk-go/service/s3"
+)
+
+func TestS3Backend(t *testing.T) {
+
```

Comment:
Something like this: https://github.com/hashicorp/vault/blob/master/physical/consul_test.go#L13 should be done to skip the test if we do not have the credentials provided.